src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -341,6 +341,22 @@ public string GetHelpMaml(string helpItemName, string path)
         #endregion
 
         #region CmdletProvider members
+        // OneDrive placeholder support (issue #8315)
+#if !UNIX
+        [DllImport("ntdll.dll")]
+        static extern sbyte RtlQueryProcessPlaceholderCompatibilityMode();
+        [DllImport("ntdll.dll")]
+        static extern sbyte RtlSetProcessPlaceholderCompatibilityMode(sbyte pcm);
```

Comment:
Should the type be char?
https://docs.microsoft.com/en-us/cpp/dotnet/calling-native-functions-from-managed-code?view=vs-2017
```suggestion
        internal static extern char RtlSetProcessPlaceholderCompatibilityMode(char pcm);
```

Also please use `internal static`.